sdk/include/asm/macamd64.inc

```diff
@@ -75,52 +75,52 @@ MACRO(BEGIN_EPILOGUE)
 ENDM
 
 MACRO(LEAF_ENTRY, Name, Section, NoPad)
-    Section segment para 'CODE'
-    ifb <NoPad>
-        db 6 dup HEX(CC)
-    endif
-    align 16
-    public  Name
-Name    proc    frame
+//Section segment para 'CODE'
+//ifb <NoPad>
+//    db 6 dup (0cch)
+//endif
```

Comment:
In the corresponding commit you say "Fix the entry points a bit" without much more details.
Why did you have to disable this code -- or in other words, what's happening to this NoPad case? (but since you keep it, this means it may be needed again later....)

Reply:
It's the minimal version of what works on GAS. It can probably be improved, but for now it should be good enough.
I'll add more details to the commit message.